wikipedia-dark.user.css

```diff
@@ -37,21 +37,21 @@
 @preprocessor uso
 ==/UserStyle== */
 @-moz-document domain("wikipedia.org"),
-domain("wikidata.org"),
-domain("mediawiki.org"),
-domain("wikitech.wikimedia.org"),
-domain("otrs-wiki.wikimedia.org"),
-domain("commons.wikimedia.org"),
-domain("meta.wikimedia.org"),
-domain("species.wikimedia.org"),
-domain("wiktionary.org"),
-domain("wikivoyage.org"),
-domain("wikiversity.org"),
-domain("wikinews.org"),
-domain("wikiquote.org"),
-domain("wikisource.org"),
-domain("wikibooks.org"),
-domain("wiki.rpcs3.net") {
+               domain("wikidata.org"),
+               domain("mediawiki.org"),
+               domain("wikitech.wikimedia.org"),
+               domain("otrs-wiki.wikimedia.org"),
+               domain("commons.wikimedia.org"),
+               domain("meta.wikimedia.org"),
+               domain("species.wikimedia.org"),
+               domain("wiktionary.org"),
+               domain("wikivoyage.org"),
+               domain("wikiversity.org"),
+               domain("wikinews.org"),
+               domain("wikiquote.org"),
+               domain("wikisource.org"),
+               domain("wikibooks.org"),
+               domain("wiki.rpcs3.net") {
```

Comment:
This maybe unnecessary adding so many white spaces here See [Style Guide](https://github.com/StylishThemes/Wikipedia-Dark/blob/master/CONTRIBUTING.md#style-guide) in contributing.md, This may throw perfectionist into a fit, idk will test it out later)

Reply:
Actually perfectionist didn't throw a fit, I think however this would look more in the styleguide as this below.
```css
@-moz-document
  domain("wikipedia.org"),
  domain("wikidata.org"),
  domain("mediawiki.org"),
  domain("wikitech.wikimedia.org"),
  domain("otrs-wiki.wikimedia.org"),
  domain("commons.wikimedia.org"),
  domain("meta.wikimedia.org"),
  domain("species.wikimedia.org"),
  domain("wiktionary.org"),
  domain("wikivoyage.org"),
  domain("wikiversity.org"),
  domain("wikinews.org"),
  domain("wikiquote.org"),
  domain("wikisource.org"),
  domain("wikibooks.org"),
  domain("wiki.rpcs3.net") {
```

Reply:
@the-j0k3r That would probably be better, yes, but perfectionist was the one that put them there.
It seems to be adhering to the practice of lining up arguments to a function for readability:
```python
def my_function_name(arg1=None, arg2='Foo', bar=Baz,
                     yet_another_arg='Something else',
                     one_more_arg=42)
```
I'm not sure how to override it. :/

Reply:
Although, I suppose the aforementioned `regexp()` syntax would obviate the need to fix this.

Reply:
Perfectioninst wont affect this part, though for readability what I suggested feels more inline with the style guide and overall indenting.

Reply:
Hmm, every time I tried to put it back, `npm run clean` indented it again. I'll convert it to a regex then. How about this one?
```
regexp(https?:\/\/[^?]*((wikipedia|mediawiki|wikibooks|wikidata|wikinews|wikiquote|wikisource|wikiversity|wikivoyage|wiktionary)|(meta|commons|otrs-wiki|species|wikitech)\.wikimedia)\.org\/.+)
```

Reply:
>Hmm, every time I tried to put it back, npm run clean indented it again

That doesnt happen here at all. I always run it before doing a release.

> I'll convert it to a regex then. How about this one?

regex is Chinese to me, I admit defeat in there. So whatever works. =)

Reply:
Regex it is then. :)